client/app/license/license.html

```diff
@@ -5,40 +5,35 @@
     <div class="page-header">
         <div class="row">
             <div class="col-sm-9">
-                <h1 data-translate="license_dashboard_title"></h1>
-            </div>
-            <div class="col-sm-3 d-flex">
-                <div class="btn-group ml-auto mt-4">
-                    <button type="button"
-                            class="oui-button oui-button_dropdown mb-0"
-                            aria-haspopup="true"
-                            aria-expanded="false"
-                            data-toggle="dropdown">
-                        <span data-translate="common_actions"></span>
-                        <span class="oui-icon oui-icon-chevron-down" aria-hidden="true"></span>
-                    </button>
-                    <ul class="dropdown-menu dropdown-menu-right m-0 p-0">
-                        <li>
-                            <a class="btn btn-link"
-                               data-ui-sref="app.license.order"
-                               data-translate="license_order_title_button">
-                            </a>
-                        </li>
-                        <li>
-                            <button type="button"
-                                    class="btn btn-link"
-                                    data-ng-click="setAction('spla/add/license-spla-add')"
-                                    data-translate="license_spla_add_title_button">
-                            </button>
-                        </li>
-                    </ul>
-                </div>
+                <h1 class="oui-header_2"
+                    data-translate="license_dashboard_title"></h1>
             </div>
         </div>
     </div>
+
     <div class="tab-content">
-        <h2 data-translate="license_dashboard_subtitle"></h2>
-        <div data-ovh-alert></div>
+        <div class="row d-md-flex">
+            <div class="col-md-6">
+                <h2 class="oui-header_3"
+                    data-translate="license_dashboard_subtitle"></h2>
+                <div data-ovh-alert></div>
+                <div class="oui-action-menu-container">
+                    <oui-action-menu aria-label="{{ 'common_actions' | translate }}"
```

Comment:
{{:: 'common_actions' | translate }}
please add one time binding everywhere for static translations